sig-docs/charter.md

```diff
@@ -0,0 +1,97 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io. Kubernetes documentation includes:
+  - Documentation of the core Kubernetes APIs
+  - Core Kubernetes architecture 
+  - CLI tools shipped with the Kubernetes release
+
+ Responsibility for creating feature documentation belongs to the developers and SIGs associated with a feature. This includes documentation for features, concepts, components, opinions, processes, etc.
+
+SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and [coordinates documentation updates][docs-release] during quarterly releases.
+
+SIG Docs maintains the Kubernetes website's infrastructure and tooling.
+
+SIG Docs creates subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website:
+
+  - Infrastructure and tooling
+  - Support and standards for internationalization (i18n) and localization (L10n)
+  - Maintaining and applying the style guide to feature documentation and release notes
+  - Reviewing and publishing release notes during quarterly release cycles
+  - Generated reference documentation for:
+    - The core Kubernetes APIs
+    - Kubernetes command-line tools, including but not limited to kubectl
+    - Kubernetes setup tools
+    - The Federation API
+
+See Cross-cutting for limitations on how the SIG helps with related efforts on other projects.
+
+#### Cross-cutting and Externally Facing Processes
+
+- [Standards for content][standards]: style guide, contributor guide, PR reviews
+  SIG Docs is not responsible for branding any content related to the Kubernetes organization. We are sometimes informally invited to consult on branding and design decisions, but we have no formal responsibility for such work.
+  
+- [Coordinating docs contributions for quarterly releases][docs-release]
+
+- Kubernetes blog:
+  SIG Docs hosts the Kubernetes blog and provides tooling and workflow support for publishing the blog, but does not directly review blog posts or work with blog contributors.
+
+- Site UX:
+  SIG Docs organizes and revises technical content to improve UX and technical accuracy for the current and four most previous releases, based on:
+  - user-reported issues
+  - site analytics
+  - content audits performed by SIG members
+
+- Other project documentation:
+  SIG Docs is sometimes advises on Kubernetes components outside of the Kubernetes/Kubernetes GitHub repository, or on other Kubernetes subprojects, but is not responsible for the documentation of any of these components or projects.
+
+### Out of scope
+
+- SIG Docs is not responsible for documenting new technical content.
```

Comment:
I think this sentence could be read a few ways that aren't correct :)

Sig Docs is not responsible for documenting new technical features to Kubernetes. We are responsible for supplementary content, such as overviews and tutorials that help onboard new users.

Reply:
Personally, I think you should strike this sentence. It's confusing.